starter-code/src/clue.js

```diff
@@ -84,11 +84,188 @@ name: Guest House
+  last_name:    "Orchid",
+  color:        "white",
+  description:  "PhD in plant toxicology. Adopted daughter of Mr. Boddy",
+  age:          26,
+  image:        "http://www.radiotimes.com/uploads/images/Original/111967.jpg",
+  ocupation:   "Scientist"
+};
+
+var profPlum = {
+  first_name:   "Victor",
+  last_name:    "Plum",
+  color:        "purple",
+  description:  "Billionare video game designer",
+  age:          22,
+  image:        "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg",
+  occupation:   "Designer"
+};
+
+var missScarlet = {
+  first_name:   "Kasandra",
+  last_name:    "Scarlet",
+  color:        "red",
+  description:  "She is an A-list movie star with a dark past",
+  age:          31,
+  image:        "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+  occupation:   "Actor"
+};
+
+var mrsPeacock = {
+first_name:   "Eleanor",
+last_name:    "Peacock",
+color:        "blue",
+description:  "She is from a wealthy family and uses her status and money to earn popularity",
+age:          36,
+image:        "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+occupation:   "Socialité"
+};
+
+var mrMustard = {
+first_name:   "Jack",
+last_name:    "Mustard",
+color:        "yellow",
+description:  "He is a former football player who tries to get by on his former glory",
+age:          62,
+image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+occupation:   "Retired Football player"
+};
+
+var rope = {
+  name: "rope",
+  weight: 10
+};
+
+var knife = {
+name: "knife",
+weight: 8,
+};
+
+var candlestick = {
+name: "candlestick",
+weight: 2
+};
+
+var dumbbell = {
+  name: "dumbbell",
+  weight: 30
+};
+
+var poison = {
+  name: "poison",
+  weight: 2
+};
+      
+var axe = {
+  name: "axe",
+  weight: 15
+};
+ var bat = {
+  name: "bat",
+  weight: 13
+ };
+
+ var trophy = {
+  name: "trophy",
+  weight: 25
+ };
+
+ var pistol = {
+  name: "pistol",
+  weight: 20
+ };
+
+/*
+var Rooms = [
+  "Dinning Room",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "Billiard Room",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "Living Room",
+  "Observatory",
+  "Theater",
+  "Guest House",
+  "Patio"
+]
+*/
 // Characters Collection
-var charactersArray = [];
+var charactersArray = [
+  mrGreen,
+  drOrchid,
+  profPlum,
+  missScarlet,
+  mrsPeacock,
+  mrMustard
+];
+
 
 // Rooms' Collection
-var roomsArray = [];
+var roomsArray = [
+  "Dinning Room",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "Billiard Room",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "Living Room",
+  "Observatory",
+  "Theater",
+  "Guest House",
+  "Patio"
+];
+
 
 // Weapons Collection
-var weaponsArray = [];
+var weaponsArray = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
+
+function randomSelector (array) {
+  var randomNumber = Math.floor((Math.random() * array.length) );
```

Comment:
```
var random = Math.floor(Math.random() * (array.length + 1));
```
Otherwise it will not be able to pick randomly the last object.

Reply:
Oops! Bad correction! If you are using the number as an index it's okay :disappointed: sorry